src/OpenApi/Schemas/Models/MemberSchema.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace CSlant\Blog\Api\OpenApi\Schemas\Models;
+
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+
+#[Schema(
+    schema: "Member",
+    required: ["id", "email"],
+    properties: [
+        new Property(property: "id", type: "integer", uniqueItems: true),
+        new Property(property: "first_name", description: "First name", type: "string", maxLength: 120, example: "Francis"),
+        new Property(property: "last_name", description: "Last name", type: "string", maxLength: 120, example: "Duong"),
+        new Property(property: "email", type: "string", format: "email", maxLength: 120, example: "francisduong@gmail.com"),
+        new Property(property: "phone", type: "string", example: "0904064751"),
```

Comment:
please Noooooooo!! Don't share your contact on the public package!!

```suggestion
        new Property(property: "phone", type: "string"),
```